Orange-Skeletons/welcome-to-nashville
# Jg parks drop down

# Description

added the dropdown in the DOM, also I added a slice method on the search formMethod 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-parks-drop-down`
4. when selecting a value in the drop down, there should be no more than 5 results that display on the DOM

# Checklist:
